src/inmanta/execute/runtime.py

```diff
@@ -20,7 +20,9 @@
 from collections.abc import Hashable, Sequence, Set
 from typing import TYPE_CHECKING, Deque, Generic, List, Literal, NewType, Optional, TypeVar, Union, cast
 
+import inmanta.ast
```

Comment:
We already use this one partially qualified as `ast` here (three lines down). We should stay consistent.